in_toto/gpg/constants.py

```diff
@@ -0,0 +1,56 @@
+"""
+<Module Name>
+  gpg/constants.py
+
+<Author>
+  Santiago Torres-Arias <santiago@nyu.edu>
+
+<Started>
+  Nov 15, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  aggregates all the constant definitions and lookup structures for signature
+  handling
+"""
+import in_toto.gpg.rsa as rsa
+import in_toto.gpg.dsa as dsa
+
+GPG_SIGN_COMMAND = "gpg2 --detach-sign --digest-algo SHA256 {keyarg} {homearg}"
+GPG_EXPORT_PUBKEY_COMMAND = "gpg2 {homearg} --export {keyid}"
+GPG_VERSION_COMMAND = "gpg2 --version"
+
+FULLY_SUPPORTED_MIN_VERSION = "2.1.0"
+
+PACKET_TYPES = {
+    'signature_packet': 0x02,
+    'main_pubkey_packet': 0x06,
+}
+
+SUPPORTED_SIGNATURE_PACKET_VERSIONS = {0x04}
+SUPPORTED_PUBKEY_PACKET_VERSIONS = {0x04}
+
+SUPPORTED_SIGNATURE_ALGORITHMS = {
+    0x01: {
+      "type":"rsa", 
+      "method": "pgp+rsa-pkcsv1.5", 
+      "handler": rsa
+    },
+    0x11: {
+      "type": "dsa",
+      "method": "pgp+dsa-fips-180-2",
+      "handler": dsa
+  }
```

Comment:
I think `common.py` has an exception message stating that `0x01` is the only supported signature algorithm.  Why is `0x11` listed here?

Reply:
Where did you see that exception message?

Reply:
I think it was this comment:

https://github.com/SantiagoTorres/in-toto/blob/778d10690dba95d67886f9a546a8778e1342ae30/in_toto/gpg/common.py#L81

There are also instances of "canonical signature type."  I'm not sure most readers will know what is meant by that term.

Reply:
Hope 1b620a1bc86c9da04e047b039c73aae980e464c2 addressed this one